### PR TITLE
Various fixes

### LINKED ISSUE #4880: Gaierror Episode 5: Sentry Edition
```python
Traceback (most recent call last):
File "TriblerGUI\event_request_manager.py", line 114, in on_read_data
RuntimeError: Unhandled Error
Traceback (most recent call last):
File "lib\site-packages\twisted\python\log.py", line 101, in callWithLogger

File "lib\site-packages\twisted\python\log.py", line 84, in callWithContext

File "lib\site-packages\twisted\python\context.py", line 118, in callWithContext

File "lib\site-packages\twisted\python\context.py", line 81, in callWithContext

--- <exception caught here> ---
File "lib\site-packages\twisted\internet\selectreactor.py", line 149, in _doReadOrWrite

File "lib\site-packages\twisted\internet\udp.py", line 227, in doRead

socket.gaierror: [Errno 10022] getaddrinfo failed
```
Report [27434](http://reporter.tribler.org/stacktrace/27434)
Difficult to reproduce.

### LINKED ISSUE #5564: [7.5.2] asyncio.exceptions.TimeoutError during version check
As reported on Windows (report 34877 and 34909):

```
Traceback (most recent call last):
File "src\tribler-gui\tribler_gui\event_request_manager.py", line 116, in on_read_data
RuntimeError: 
--LONG TEXT--
Traceback (most recent call last):
File "src\tribler-core\tribler_core\modules\versioncheck_manager.py", line 45, in check_new_version_api
File "lib\site-packages\aiohttp\client.py", line 582, in _request
File "lib\site-packages\aiohttp\helpers.py", line 596, in __exit__
asyncio.exceptions.TimeoutError
--CONTEXT--
{'message': 'Task exception was never retrieved', 'exception': TimeoutError(), 'future': <Task finished name='Task-208843' coro=<VersionCheckManager.check_new_version_api() done, defined at src\tribler-core\tribler_core\modules\versioncheck_manager.py:42> exception=TimeoutError()>}
```

### PR DESCRIPTION
**Pull request by devos50**
Fixes #5564 , fixes #4880

### CODE REVIEW

**Comment by devos50** [comment]
retest this please

**Comment by devos50** [comment]
Ready for review. The lines reported by the coverage check are non-trivial to cover with unit tests.

File: src/tribler-gui/tribler_gui/core_manager.py [hunk 1]
```diff
@@ -59,7 +60,8 @@ def on_core_finished(self, exit_code, exit_status):
             exception_msg = "The Tribler core has unexpectedly finished with exit code %s and status: %s!" % \
                             (exit_code, exit_status)
             if self.core_traceback:
-                exception_msg += "\n\n%s" % self.core_traceback
+                exception_msg += "\n\n%s\n(Timestamp: %d, traceback timestamp: %d)" % \
+                                 (self.core_traceback, int(round(time.time() * 1000)), self.core_traceback_timestamp)
```
Comment (xoriole) [inline src/tribler-gui/tribler_gui/core_manager.py:64]: where is `self.core_traceback_timestamp` coming from?

**Review by qstokkink — APPROVED** [review]

File: src/tribler-gui/tribler_gui/core_manager.py (hunk 1 above)
Comment (devos50) [reply src/tribler-gui/tribler_gui/core_manager.py:64]: Oh whoops! Seems that I removed one variable too much :'(

**Review by xoriole — APPROVED** [review]

**Review by qstokkink — APPROVED** [review]

**Pull request merged**